index.html

```diff
@@ -209,6 +189,18 @@ <h2 class="outro2">LET'S CONNECT</h2>
         </div>
       </div>
     </section>
+  <section class="contactme">
+    <h1 class="contacttxt font">Contact me</h1>
+    <p class="paragraph font">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/{form_id}" method="post" class="">
+    <input type="text" name="name" id="name" maxlength="30" placeholder="Enter your name" class="inputtxt1 radius">
+    <br>
+    <input name="Email" id="email" type="email" placeholder="Enter your email address" class="inputtxt2 radius">
+    <br>
+    <textarea name="textarea" id="message" placeholder="Write your message here" cols="30" rows="10" class="inputtxt3 radius"></textarea>
+    </form>
+    <button class="button3" type="submit" name="submit" value="submit"><p class="button3txt radius">Get in touch</p></button>  
```

Comment:
- [ ]  You did a great job working on this milestone! 💯 However, you are not using the formspree service the way it is expected, in your form action, you have the `{form_id}` at the end. This value uses curly braces because it's supposed to be dynamic, this form_id should be the `id` you created in [formspree](https://formspree.io/html/) so please consider fixing this and testing it before submitting your code again! 🤓 🔧

- [ ] Kindly, It could be better if you remove the `ul`, `li`, and `<br>` tags in your `form` , then you can use the `Flexbox` in order to lay out the elements on the page as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20on%20the%20page%20you%20should%20use%20Flexbox.) 👍. You my use this form : 
````
<form>
      <input />
      <input />
      <textarea></textarea>
      <button></button>
</form>
````

- [ ] Please, add in All input fields the `required` attribute to activate the validation on submit. It'll be something like this : 
````
 <input type="text" maxlength="30" placeholder="Full name" name="userName" required/>
`````

- [ ] Also, make sure your `<textarea>` (Message field)  accepts up to 500 chars (`maxlength="500"`)

- [ ] Kindly, make sure you implemented the button interactions (enable, hover, etc.) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=Implement%20the%20button%20interactions%20(enable%2C%20hover%2C%20etc.).)

![Annotation 2023-02-23 152637](https://user-images.githubusercontent.com/57408419/220920616-3204329f-5e31-4dc2-8154-f20e9308054a.jpg)

Reply:
Done